packages/serverless-collector/README.md

```diff
@@ -0,0 +1,22 @@
+# @instana/serverless-collector &nbsp; [![OpenTracing Badge](https://img.shields.io/badge/OpenTracing-enabled-blue.svg)](http://opentracing.io)
+
+Monitor your Node.js applications on **any** serverless environment with Instana!
+
+**Note:** This package is currently under development and marked as **beta**. Before you proceed with updating to a newer version, please ensure to review the release notes for any upcoming changes that could potentially cause compatibility issues.
+
+## Restrictions
+
+The serverless collector **does not** support autotracing and metrics.
+
+## Installation
+
+```sh
+npm i @instana/serverless-collector --save
```

Comment:
Note:  When we add a new package to npm we manually need to addnpackage to our Organization before the release.